database/migrations/2020_08_24_130222_alter_users_table.php

```diff
@@ -33,7 +33,9 @@ public function down()
     {
         Schema::table('users', function (Blueprint $table) {
             //$table->dropColumn('gapi_access_token');
-            $table->string('gapi_access_token')->nullable()->default(null)->change();
+            // don't rollback this because it'll throw exception if column has values greater than default (255) varchar
+            // either specify the large length
+            $table->string('gapi_access_token', 10000)->nullable()->default(null)->change();
```

Comment:
Is it better to use string instead of text?

Reply:
It's down function, I think text is better instead of defining large chars length. I'll update this.

Reply:
Done